src/BugsnagUnity/Native/Android/NativePayloadClassWrapper.cs

```diff
@@ -267,7 +267,7 @@ public void SetNativeMetadataSection(string key, string section, Dictionary<stri
             return date;
```

Comment:
I think we should construct a `DateTimeOffset` on the line above rather than relying on an implicit cast.

Reply:
Done 👍